deploy/examples/common.yaml

```diff
@@ -84,7 +84,17 @@ metadata:
     app.kubernetes.io/name: cosi-driver-ceph
 rules:
   - apiGroups: ["objectstorage.k8s.io"]
-    resources: ["buckets", "bucketaccesses", "bucketclaims", "bucketaccessclasses", "buckets/status", "bucketaccesses/status", "bucketclaims/status", "bucketaccessclasses/status"]
```

Comment:
Why did you modify this line?

Reply:
This was changed when it was auto-generated, but I'm not sure why it was generated differently..
I will fix it with the original code